src/Symfony/Component/BrowserKit/Client.php

```diff
@@ -162,6 +162,17 @@ public function getServerParameter($key, $default = '')
         return isset($this->server[$key]) ? $this->server[$key] : $default;
     }
 
+    /**
+     * @param string      $method
+     * @param string      $uri
+     * @param array       $parameters
+     * @param array       $files
+     * @param array       $server
+     * @param string|null $content
+     * @param bool        $changeHistory
```

Comment:
this is adding no value compared to the method signature. It only repeats info already in it. So omitting it is done on purpose.